README.md

```diff
@@ -14,26 +14,34 @@ and is available for use with the following clients.
 
 ### Completion for Liberty server configuration files
 * Completion for Liberty properties and values 
-![Completion for Liberty properties](./docs/images/property-completion.png) 
-![Completion for Liberty property values](./docs/images/property-value-completion.png)
+
+![Completion feature for Liberty properties in bootstrap.properties](./docs/images/property-completion.png) 
```

Comment:
Perhaps add a sentence other than the label such as:
When editing a bootstrap.properties file and entering a property name, completion support displays a list of potential property names that match what has been entered.

That was off the top of my head. Feel free to reword. For values, you can also mention how if a default value is defined, it is at the top of the list and is preselected.

Reply:
So for each image, there should be a sentence or two that actually describes what the user would see.